driverthroughput/main.py

```diff
@@ -286,6 +324,7 @@ def _create_colocated_throughput_session(cls,
 
         name = "-".join((
             "throughput-sess-colo",
+            str(language),
```

Comment:
if language is already "cpp" or "fortran", you don't need to convert it to a string, right?